File: talleres javascrip/interaccionusuario/usuario.js
function obtener() {
            let a = prompt('Escriba sus nombres completos:');
            let b = prompt('Escriba sus apellidos completos:');
            let c = prompt('Escriba su profesión:');
            let d = parseInt(prompt('Escriba su año de nacimiento:'));
            let e = 2025 - d;
            
            alert('El señor(@)'+ a + ' trabaja en ' + c + ' ' + b + ' y el tiene ' + e + ' años');
        }

File: GO/usuario.go
package main

import "fmt"

func main() {
	// Definimos variables para los datos de entrada
	var a, b, c string
	var d, e int

	// Solicitamos los datos al usuario
	fmt.Print("Escriba sus nombres completos: ")
	fmt.Scanln(&a)

	fmt.Print("Escriba sus apellidos completos: ")
	fmt.Scanln(&b)

	fmt.Print("Escriba su profesion: ")
	fmt.Scanln(&c)

	fmt.Print("Escriba su año de nacimiento: ")
	fmt.Scanln(&d)

	// Calculamos la edad
	e = 2025 - d

	// Imprimimos el resultado
	fmt.Printf("El (La) %s %s %s tiene %d años\n", c, a, b, e)
}
